src/containers/pages/FocusInvestigation/active/index.tsx

```diff
@@ -269,7 +270,7 @@ class ActiveFocusInvestigation extends React.Component<
                     Cell: (cell: CellInfo) => {
                       return (
                         <div>
-                          {cell.original.focusArea.trim() && (
+                          {cell.value.trim() && (
```

Comment:
@p-netm why this change?

Reply:
An old change from when i had tried replacing ' NULL' jurisdiction ids with an empty string, it would stop displaying the the plan title after changing the focusArea value to an empty string. I can revert this.